hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/UsageBasedFeeCalculatorTest.java

```diff
@@ -346,21 +345,16 @@ void throwsIseOnBadScheduleInFcfs() {
     }
 
     @Test
-    void failsWithIseGivenApplicableButUnusableCalculator() throws InvalidTxBodyException {
+    void failsWithIseGivenApplicableButUnusableCalculator() {
         // setup:
         final SigValueObj expectedSigUsage =
                 new SigValueObj(FeeBuilder.getSignatureCount(signedTxn), 9, FeeBuilder.getSignatureSize(signedTxn));
 
         given(correctOpEstimator.applicableTo(accessor.getTxn())).willReturn(true);
         given(txnUsageEstimators.get(CryptoCreate)).willReturn(List.of(correctOpEstimator));
-        given(correctOpEstimator.usageGiven(
-                        argThat(accessor.getTxn()::equals),
-                        argThat(factory.apply(expectedSigUsage)),
-                        argThat(view::equals)))
-                .willThrow(InvalidTxBodyException.class);
 
         // when:
-        assertThrows(IllegalArgumentException.class, () -> subject.computeFee(accessor, payerKey, view, consensusNow));
+        assertThrows(NullPointerException.class, () -> subject.computeFee(accessor, payerKey, view, consensusNow));
```

Comment:
Is this correct? This looks more like a mock failed to return something

Reply:
Added a new mock. But NPE is expected